src/test/scala/com/horizon/exchangeapi/BusinessSuite.scala

```diff
@@ -207,7 +207,7 @@ class BusinessSuite extends AnyFunSuite {
     val input = PostPutBusinessPolicyRequest(businessPolicy, Some("desc"),
       BService(svcurl, orgid, svcarch, List(BServiceVersions(svcversion, Some(Map("priority_value" -> 50)), Some(Map("lifecycle" -> "immediate")))), Some(Map("check_agreement_status" -> 120)) ),
       Some(List( OneUserInputService(orgid, svcurl, None, None, List( OneUserInputValue("UI_STRING","mystr"), OneUserInputValue("UI_INT",5), OneUserInputValue("UI_BOOLEAN",true) )) )),
-      Some(List( OneSecretBindingService(orgid,svcurl, None, None, List(OneSecretBindingValue("secret1",""))))),
+      Some(List( OneSecretBindingService(orgid,svcurl, None, None, List(Map("secret1"->""))))),
```

Comment:
You should also give a non-blank value, in addition to the key. That way you can check that it gets set correctly. Note that this is a mapping the service secret name to the vault secret name, so a good example, would be: `Map("service-secret1"->"vault-secret1")`

This comment also applies to all of the similar lines below.

Also add tests in which you do a GET and verify that the value was set correctly (for POST, PUT, and PATCH)